rcp/org.testeditor.rcp4.uatests/src/main/java/org/testeditor/fixture/eclipse/EclipseWorkbenchFixture.java

```diff
@@ -80,6 +94,20 @@ public void closeEditor(IEditorPart editorPart) {
 		syncExec(() -> window.getActivePage().closeEditor(editorPart, false));
 	}
 
+	/**
+	 * Wait until their is no running job in the background of the RCP/IDE.
+	 */
+	@FixtureMethod
+	public void waitUntilJobsCompleted() {
+		try {
+			while (!Job.getJobManager().isIdle()) {
+				Thread.sleep(100);
```

Comment:
There is no time-out here. If we got a job that is running forever the tests will stall.